Move /usage/distributed-tracing to /tracing (#9572)
getsentry/sentry-docs: docs/platforms/flutter/tracing/index.mdx

---
title: Tracing

> description: "Learn how to connect events across applications/services."

Reviewer: ```suggestion
description: "Learn how to connect events across multiple applications and services using tracing."
```

sidebar_order: 40
---

> If the overall application landscape that you want to observe with Sentry consists of more than just a single service or application, distributed tracing can add a lot of value.

Reviewer: ```suggestion
Tracing allows you to see interactions across multiple systems. If your application relies on more than one service, setting up tracing can help catch issues earlier and faster by providing a holistic view of your application landscape.
```

> ## What is Tracing?

Reviewer: ```suggestion
## What Is Tracing?
```

> In the context of a distributed system, tracing acts as a powerful debugging tool. Imagine your application as a vast network of interconnected parts. For example, your system might be spread across different servers or your application might split into different backend and frontend services, each potentially having their own technology stack.
> 
> When an error or performance issue occurs, it can be challenging to pinpoint the root cause due to the complexity of such a system. Tracing helps you follow the path of an event as it travels through this intricate web, recording every step it takes. By examining these traces, you can reconstruct the sequence of events leading up to the event of interest, identify the specific components involved, and understand their interactions. This detailed visibility enables you to diagnose and resolve issues more effectively, ultimately improving the reliability and performance of your distributed system.

Reviewer: ```suggestion
If you're working within a distributed system where your application is a vast network of interconnected parts, tracing can be a powerful debugging tool.

Trying to pinpoint the root cause of an error or performance issue in a system that's spread across different servers, or split into different backend and frontend services (each with their own technology stack), can be challenging.

Tracing helps you follow the path of an event as it travels through this intricate web, recording every step it takes. By looking at traces, you can reconstruct the sequence of events that lead up to the event of interest, identify the components involved, and understand their interactions. This detailed visibility lets you diagnose and resolve issues more effectively and improve the overall reliability and performance of your distributed system.
```

## Basic Example

Here's an example showing a trace in Sentry:

<Include name="common-imgs/distributed-trace-in-sentry" />

This trace shows a Vue app's `pageload` making a request to a Python backend, which then calls the `/api` endpoint of a Ruby microservice.

> What happens in the background is that Sentry uses reads and further propagates two HTTP headers between your applications:

Reviewer: ```suggestion
In the background, Sentry uses reads and further propagates two HTTP headers between your applications:
```
